server/controller/src/main/java/ai/starwhale/mlops/domain/dataset/dataloader/DataReadManager.java

```diff
@@ -106,45 +195,48 @@ public Session generateSession(DataReadRequest request) {
                                 .status(Status.DataStatus.UNPROCESSED)
                                 .build())
                         .collect(Collectors.toList()),
-                1000).forEach(dataReadLogDao::batchInsert);
-
-        return session;
-
+                this.insertBatchSize
```

Comment:
list may be empty, and the `batchInsert` may fail.

Reply:
done